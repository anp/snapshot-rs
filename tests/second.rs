mod test {
    use snapshot::snapshot;

    #[snapshot]
    fn another_simple_snapshot() -> i32 {
        let x = 2;
        x
    }
}
