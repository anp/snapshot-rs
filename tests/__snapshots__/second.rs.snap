{
  "second::test::another_simple_snapshot": {
    "file": [
      "tests",
      "second.rs"
    ],
    "module_path": "second::test",
    "test_function": "another_simple_snapshot",
    "recorded_value": 2
  }
}